helm-chart/renku/values.yaml

```diff
@@ -22,7 +22,6 @@ global:
     # url:
```

Comment:
@jachro an unruly formatter hit again!

Reply:
That's not me :) I didn't even open the file.
But I'll fix it.

Reply:
Actually, it happened in this commit `ae26f19856385c6880d8266fae361dc2d6481b35` which was done by the Bot. I'm guessing the script updating the version might be doing some reformatting.

Reply:
ah, yes, sorry, now I remember. It's something to do with `yq`... I think the upgrade should fix it and it's in my TODO...

Reply:
@aledegano I fixed it.

Reply:
Thanks Kuba!
I'll check the problem with the script.